docs/book/how-to/stack-deployment/deploy-a-cloud-stack.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
It would make sense to also add something like this like I added in the blog:

![image](https://github.com/zenml-io/zenml/assets/4792082/caadbfe4-3bdd-4cc3-9cb0-20e3974f44a6)